FA.Cache/Providers/RedisCacheProvider.cs

```diff
@@ -8,11 +8,14 @@ namespace FA.Cache.Providers
     public class RedisCacheProvider : ICacheProvider
     {
         private readonly IDatabase _cache;
+        private readonly IServer _server;
 
         public RedisCacheProvider(string redisConnectionString)
         {
             Console.WriteLine("RedisCacheProvider: Setting up redis connection...");
             var redis = ConnectionMultiplexer.Connect(ConfigurationOptions.Parse(redisConnectionString));
+            //Assuming only one server
+            _server = redis.GetServers().FirstOrDefault();
```

Comment:
We sure to use this way only? Are we sure this is official way?
any supportive link?

Reply:
https://stackexchange.github.io/StackExchange.Redis/KeysScan#so-how-do-i-use-them

Reply:
Or we can use Lua based modifications